src/main/java/com/swisscom/ais/TestOnDemandSignatureWithStepUp.java

```diff
@@ -52,6 +55,10 @@ public static void main(String[] args) throws Exception {
             document.setInputFromFile(properties.getProperty("local.test.inputFile"));
             document.setOutputToFile(properties.getProperty("local.test.outputFilePrefix") + System.currentTimeMillis() + ".pdf");
             document.setDigestAlgorithm(DigestAlgorithm.SHA512);
+            
+            // to create a visible signature in the pdf, specify the signature definition and the page. Optionally, provide an icon that will be embedded in the visual signature
+            document.setVisibleSignatureDefinition(new VisibleSignatureDefinition(200, 200, 150, 30, 0, new File(properties.getProperty("local.test.visibleSignatureFile"))));

```

Comment:
I propose here to take the icon file as a string (the file name) not a direct File object. This is the norm of the API for all the other classes. Also, it would be nice to add a no-args constructor to VisibleSignatureDefinition, so that I (as a user) can create an object separately and call setters for each attribute. This full-args constructor works, but it might be difficult to keep in mind what each int argument means (e.g. 200, 200, 150, 30, 0 <= what each one means?)